bioblend/_tests/TestGalaxyQuotas.py

```diff
@@ -52,3 +55,29 @@ def test_delete_undelete_quota(self):
         assert response == "Deleted 1 quotas: " + self.quota_name
         response = self.gi.quotas.undelete_quota(self.quota["id"])
         assert response == "Undeleted 1 quotas: " + self.quota_name
+
+    @test_util.skip_unless_galaxy("release_19.09")  # for user purging
+    def test_update_non_default_quota(self):
+        """
+        test if a non default quota can be updated (needs to use default=None,
+        default="no" will fail)
+        """
+        if self.gi.config.get_config()["use_remote_user"]:
+            self.skipTest("This Galaxy instance is not configured to use local users")
+        new_username = test_util.random_string()
+        new_user_email = f"{new_username}@example.org"
+        password = test_util.random_string(20)
+        new_user = self.gi.users.create_local_user(new_username, new_user_email, password)
+
+        quota = self.gi.quotas.create_quota(
+            name="non_default_quota",
+            description="testing",
+            amount="100 GB",
+            operation="+",
+            in_users=[new_user["id"]],
+        )
+        self.gi.quotas.update_quota(quota["id"], default=None, amount="200 GB")
```

Comment:
should we test like this?

```suggestion
        self.gi.quotas.update_quota(quota["id"], amount="200 GB")
```

Reply:
I don't mind either way